vmdb/db/migrate/20150405141637_remove_port_config_from_container_service.rb

```diff
@@ -0,0 +1,55 @@
+class RemovePortConfigFromContainerService < ActiveRecord::Migration
+  class ContainerService < ActiveRecord::Base
+    has_many :container_service_port_configs
+  end
+
+  class ContainerServicePortConfig < ActiveRecord::Base
+  end
+
+  def up
+    create_table :container_service_port_configs do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.integer    :port
+      t.string     :target_port
+      t.string     :protocol
+      t.belongs_to :container_service, :type => :bigint
+    end
+
+    say_with_time("Moving container_service port records to container_service_port_config table") do
+      ContainerService.all.each do |service|
+        ContainerServicePortConfig.create!(
+            :ems_ref              => "#{service.ems_ref}_#{service.port}_#{service.container_port}",
+            :port                 => service.port,
+            :protocol             => service.protocol,
+            :target_port          => service.container_port,
+            :container_service_id => service.id
+        )
+        say "service '#{service.name}' updated!"
```

Comment:
We generally don't emit this much detail into the migrations, so this should be removed.